libs/qCC_db/src/ccPointCloud.cpp

```diff
@@ -2736,7 +2736,7 @@ void ccPointCloud::drawMeOnly(CC_DRAW_CONTEXT& context)
 							const ccColor::Rgb* col = m_currentDisplayedScalarField->getValueColor(pointIndex);
 							//we force display of points hidden because of their scalar field value
 							//to be sure that the user doesn't miss them (during manual segmentation for instance)
-							glFunc->glColor4ubv(col ? col->rgb : ccColor::lightGreyRGB.rgb);
+							glFunc->glColor4ubv(col ? ccColor::FromRgbToRgba(*col).rgba : ccColor::lightGrey.rgba);
```

Comment:
wouldn't it be simpler to restore the previous call to 'glColor3ubv'? (since ccColor::lightGrey has an alpha channel equal to 255 anyway).

Reply:
I thought you needed to change to 4ubv for AMD support?

Reply:
Oh no, that alignment constraints is for VBOs only. Here I just wanted to propagate the support for transparency, but this none of the inputs are 'transparent' here, it was my mistake to convert the call to glColor4ubv.

Reply:
Yeah, I would definitely prefer the glColor3ubv option, since we are potentially calling this method millions of time, and here we are basically creating a new instance of Rgba for nothing each time.